projects/RabbitMQ.Client/client/framing/BasicAck.cs

```diff
@@ -35,14 +35,10 @@
 
 namespace RabbitMQ.Client.Framing.Impl
 {
-    internal sealed class BasicAck : Client.Impl.MethodBase
+    internal readonly struct BasicAck : IOutgoingAmqpMethod
     {
-        public ulong _deliveryTag;
-        public bool _multiple;
-
-        public BasicAck()
-        {
-        }
+        public readonly ulong _deliveryTag;
+        public readonly bool _multiple;
```

Comment:
Would be good since these are public properties to `PascalCase` them. Same applies to all the other *Method structs.

Reply:
Yes, but I kept it the same for now. Can be done in a future PR.